statesman-engine/src/main/java/io/appform/statesman/engine/action/impl/HttpAction.java

```diff
@@ -110,7 +110,9 @@ public Response visitPost() throws Exception {
                 log.info("HTTP_ACTION POST Call url:{}", url);
                 val payload = actionData.getPayload();
                 Response response = client.get().post(url, payload, headers);
-                if (!response.isSuccessful()) {
+                if ((actionData.acceptableErrorCode == null
```

Comment:
Use accessors

Reply:
This condition looks weird.. what you are trying to do is:
`if((actionData.acceptableErrorCode != null && !actionData.acceptableErrorCode.contains(response.code()))
    || !response.isSuccessful())`